agent/agent.go

```diff
@@ -5,8 +5,11 @@ import (
 	"github.com/MagalixTechnologies/uuid-go"
 	"golang.org/x/sync/errgroup"
 	"os"
+	"time"
 )
 
+const AuthorizationTimeoutDuration = 2 * time.Hour
```

Comment:
This isn't used as far as I can tell

Reply:
Was testing with a smaller time and forgot to change it. Will fix it.